shanghai/main.py

```diff
@@ -11,15 +13,15 @@
 
 
 def exception_handler(loop, context):  # pylint: disable=unused-argument
-    import io
     f = io.StringIO()
     print("exception_handler context:", file=f)
     pprint(context, stream=f)
+    traceback.print_exc(file=f)
     if 'task' in context:
         context['task'].print_stack(file=f)
     elif 'future' in context:
         context['future'].print_stack(file=f)
-    current_logger.error(f.getvalue())
+    # print(f.getvalue())
```

Comment:
yea. I had problems distinguishing exceptions that got handled by the function above and exceptions that didn't get handled. So I temporarily stopped the handler outputting the handled exceptions, to focus on the others.